docs/how-to/load-keys.md

```diff
@@ -8,19 +8,24 @@ import TabItem from '@theme/TabItem';
 
 # Load signing keys
 
-You can load signing keys by:
-
-- [Creating a key configuration file]. Key configuration files can be used to load all types of signing keys supported by Web3Signer.
-- Using the [`eth2` subcommand options](../reference/cli/subcommands.md#eth2) to bulk load consensus
-  layer signing keys stored in [Azure Key Vault](#azure-key-vault), [AWS Secrets Manager](#aws-secrets-manager),
-  or [keystore files](#keystore-files).
-- Using the [`eth1` subcommand options](../reference/cli/subcommands.md#eth1) to bulk load execution
-  layer signing keys stored in [Azure Key Vault](#azure-key-vault), [AWS Key Management Service (KMS)](#bulk-load-keys)
-  or [keystore files](#keystore-files).
+Load signing keys using a [key configuration file], or bulk loading with the [`eth1` and `eth2` subcommands]. 
+Web3Signer supports loading keys with the following methods: 
+
+| Key storage                          | Key configuration file | Bulk loading with `eth1` subcommand | Bulk loading with `eth2` subcommand | 
+|--------------------------------------|:----------------------:|:-----------------------------------:|:-----------------------------------:|
+| [Keystore files]                     |           x            |                  x                  |                  x                  | 
+| **Vaults**                           | 
+| [Hashicorp Vault]                    |           x            |                                     |                                     | 
+| [Azure Key Vault]                    |           x            |                  x                  |                  x                  | 
+| [AWS Secrets Manager]                |           x            |                                     |                  x                  | 
+| [AWS KMS]                            |           x            |                  x                  |                                     | 
+| [GCP Secret Manager]                 |           ?            |                                     |                  x                  | 
```

Comment:
GCP is only supported via bulk loading and only on eth2 mode.

Reply:
Have updated.